.github/workflows/test-azureblob.yml

```diff
@@ -15,9 +15,9 @@ jobs:
         strategy:
             matrix:
                 go-version:
-                    - 1.18.x
```

Comment:
don´t forget to update the readme
<img width="630" alt="image" src="https://github.com/gofiber/storage/assets/7063188/9080e10d-b7d1-4240-ba12-21062b3a87ea">

Reply:
Oops! Will do